pkgs/by-name/an/anubis/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  stdenv,
+}:
+buildGoModule (finalAttrs: {
+  pname = "anubis";
+  version = "1.14.2";
+
+  src = fetchFromGitHub {
+    owner = "TecharoHQ";
+    repo = "anubis";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-K4VXTO8F3GTOA9jOrI0OF6CYTRaDUlUT9/HujYmnHpM=";
+  };
+
+  vendorHash = "sha256-t+E3sILEwXGkTaBtKLO2kFEntivY9fVK8o86arvMaOU=";
+
+  subPackages = [
+    "cmd/anubis"
+  ];
+
+  ldflags =
+    [
+      "-s"
+      "-w"
+      "-X=github.com/TecharoHQ/anubis.Version=v${finalAttrs.version}"
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
+      "-extldflags=-static"
```

Comment:
Is this really necessary? At least on x86_64-linux the build also succeeds without this flag and the exact same binary is produced.

Reply:
I'm not sure. Since CGO is enabled by default, I think it is unnecessary, but the upstream builds with it https://github.com/TecharoHQ/anubis/blob/main/.ko.yaml#L12

Reply:
Ok, if upstream adds this flag, we can probably just keep it